Bottom: 39f965483932865e0dd4d42437b6a102073ac6e8
Top:    04ec74a228db26899dfcbcde337cff4e8237778c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 20:18:36 +0000

Refresh of hid-make-arguments-to-the-prom

---

diff --git a/src/hid/lesstif/dialogs.c b/src/hid/lesstif/dialogs.c
index e60c182..b3cc375 100644
--- a/src/hid/lesstif/dialogs.c
+++ b/src/hid/lesstif/dialogs.c
@@ -592,7 +592,7 @@ lesstif_prompt_for (const char *msg, const char *default_string)
   xs = XmStringCreateLocalized ((char *)msg);
   stdarg (XmNlabelString, xs);
   XtSetValues (prompt_label, args, n);
-  XmTextSetString (prompt_text, default_string);
+  XmTextSetString (prompt_text, (char *)default_string);
   XmTextSetCursorPosition (prompt_text, strlen (default_string));
   wait_for_dialog (prompt_dialog);
   rv = XmTextGetString (prompt_text);
diff --git a/src/hid/lesstif/lesstif.h b/src/hid/lesstif/lesstif.h
index 54e4ced..3bc822b 100644
--- a/src/hid/lesstif/lesstif.h
+++ b/src/hid/lesstif/lesstif.h
@@ -50,7 +50,7 @@ extern void lesstif_insert_style_buttons (Widget menu);
 extern void lesstif_styles_update_values ();
 extern void lesstif_update_layer_groups ();
 extern void lesstif_update_status_line ();
-extern char *lesstif_prompt_for (char *, char *);
+extern char *lesstif_prompt_for (const char *, const char *);
 extern char *lesstif_fileselect (const char *, const char *,
 				 char *, char *,
 				 const char *, int);
